src/EnergyPlus/DataHVACGlobals.cc

```diff
@@ -346,7 +350,8 @@ namespace DataHVACGlobals {
                                             "Coil:Heating:DX:VariableSpeed",
                                             "Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed",
                                             "",
-                                            "Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl"});
+                                            "Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl",
+					    ""});
```

Comment:
The table diffs for the VRF files must be related to these changes in DataHVACGlobals, but the old code looks correct. I wonder if the code that uses these arrays isn't seeing the last entry and now that this is second-to-last, it works?

Reply:
Coil:Cooling:DX is not showing at all in the Coil Sizing Details report, so that could be related to it being the last on the list, or more likely that the new coil isn't calling the function to populate this report. Either way, the VRF table changes are good here and this report will need more attention before release.